scripts/insert_into_appointments.sql

```diff
@@ -1 +1,6 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(date, time, comment, patient_id, doctor_id) VALUES
+('August 12, 2016', '10am', 'no comment', 8, 4),
+('August 17, 2016', '1pm', 'no comment', 5, 6),
+('August 12, 2016', '10am', 'no comment', 8, 6),
+('August 17, 2016', '1pm', 'no comment', 7, 4);
```

Comment:
Not quite. Recall that we need to use the `SELECT` method in conjunction with insert. Also, what is `date` and `time` in sql? Take a look at the documentation!

Here is 1 possible solution:
```sql
-- insert appointments creating connections between doctors and patients
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT '20150812', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hilton'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20150817', '1300',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Talley')
  UNION
  SELECT '20150817', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20150817', '1300',
   (SELECT id FROM doctors WHERE surname = 'Hilton'),
   (SELECT id FROM patients WHERE surname = 'Hodge')
;
```